projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/merchandising-carousel.e2e-spec.ts

```diff
@@ -46,19 +51,21 @@ function testCategoryPage(
   merchandisingCarousel.verifyMerchandisingCarouselRendersOnCategoryPage(
     strategyRequestAlias,
     categoryCode,
-    merchandisingCarousel.DEFAULT_LANGUAGE
+    merchandisingCarousel.DEFAULT_LANGUAGE,
+    undefined,
+    containsConsentReference
   );
 }
 
 function testBrandPage(
   language: string = merchandisingCarousel.DEFAULT_LANGUAGE,
   currency: string = merchandisingCarousel.DEFAULT_CURRENCY
 ): void {
-  cy.visit(`/${language}/${currency}/${sonyBrandPagePath}`);
+  cy.visit(`/${language}/${currency}/${canonBrandPagePath}`);
```

Comment:
sometimes test fails relating to this as it tries to navigate to a page, but qualtrics failed to load. A safety net would be using a wait to make suer the are loaded.

For example, 

```
const productCode = `ProductPage&code=${sampleProduct.code}`;
  const productPage = waitForPage(productCode, 'getProductPage');
```

above can be found under checkout.ts